### ISSUE TITLE
Speed up unit tests for subdivision checks

### ISSUE THREAD

**Issue by dhermes** [dhermes/bezier#2]
Running `py.test` with durations shows the culprits

```
$ git log -1 --pretty=%H
61c8c4bee9c0b29b129c5d829e5aed262c453566
$ tox -e py27 -- --durations=6
...
1.45s call     tests/test_surface.py::TestSurface::test_subdivide_on_the_fly
0.99s call     tests/test_surface.py::TestSurface::test_subdivide_quartic_check_evaluate
0.30s call     tests/test_surface.py::TestSurface::test_subdivide_quadratic_check_evaluate
0.28s call     tests/test_surface.py::TestSurface::test_subdivide_cubic_check_evaluate
0.27s call     tests/test_surface.py::TestSurface::test_subdivide_line_check_evaluate
0.01s call     tests/test_surface.py::TestSurface::test__compute_valid_invalid_linear
...
```

## Main Culprit

`Surface.evaluate_multi()` is slow, in particular, when we use 561 points within the reference triangle

**Comment by dhermes** [comment]
Post-Script: (Fixed in 61c8c4bee9c0b29b129c5d829e5aed262c453566). The sixth longest duration is a "lie":

```
0.18s call     tests/test__base.py::TestBase::test___eq__
```

it reflects the start-up cost of importing `matplotlib` (via importing `bezier` in `TestBase._make_one`) for the first time:

```
$ python2.7 -c 'from time import time; s = time(); import base64; print(time() - s)'
0.00239682197571
$ python2.7 -c 'from time import time; s = time(); import matplotlib; print(time() - s)'
0.147956848145
$ python3.5 -c 'from time import time; s = time(); import base64; print(time() - s)'
0.0032052993774414062
$ python3.5 -c 'from time import time; s = time(); import matplotlib; print(time() - s)'
0.18233108520507812
```

Why does `test___eq__` bear this cost? Because it is the first in alphabetical order of all the tests and because all the tests import `bezier` at runtime rather than at import time.

**Comment by dhermes** [comment]
Fixed via: https://github.com/dhermes/bezier/commit/8bcec8e02dc6026d9220590653f0019ecb2500ca
